### PR TITLE
fix unilm token type, parameters & tests issues

### PR DESCRIPTION
**Pull request by fuliucansheng**

### CODE REVIEW

File: fastseq/models/unilm_hf/modeling_unilm.py [hunk 1]
```diff
@@ -761,7 +761,7 @@ def generate(self, input_ids, attention_mask, decoder_start_token_id,
             num_beams: beam size for beam search
             num_return_sequences: num for return sequence for beam search
         """
-        max_seq_length = kwargs.pop("max_gen_length", 48)
+        max_seq_length = kwargs.pop("max_length", 48)
```
Comment (feihugis) [inline fastseq/models/unilm_hf/modeling_unilm.py:764]: Does [here](https://github.com/microsoft/fastseq/tree/main/examples/unilm#setting) also need to be updated?

**Review by feihugis — COMMENTED** [review]
Thanks @fuliucansheng for the fix! It looks good to me. Just left a minor comment.

File: fastseq/models/unilm_hf/modeling_unilm.py (hunk 1 above)
Comment (fuliucansheng) [reply fastseq/models/unilm_hf/modeling_unilm.py:764]: No need to update it. The parameter for fastseq-cli is `max_gen_length`. [Here](https://github.com/microsoft/fastseq/blob/main/fastseq_cli/transformers_generate.py#L290)

**Comment by feihugis** [comment]
Triggered the CI test internally. All the tests passed. I will merge this PR.

**Pull request merged**